sdk/core/core-amqp/test/errors.spec.ts

```diff
@@ -20,21 +20,31 @@ class AMQPError {
 
 describe("Errors", function() {
   describe("translate", function() {
-    it("Converts to MessagingError, and acts as a passthrough if the input is not an AmqpProtocolError", function() {
-      const MyError: any = function() {};
-      const err: any = new MyError();
-      const msg: any = undefined;
-      const ehError = new Errors.MessagingError(msg);
-      const translatedError = <Errors.MessagingError>Errors.translate(err);
-      translatedError.name.should.equal(ehError.name);
-      translatedError.retryable.should.equal(ehError.retryable);
-      translatedError.message.should.equal(ehError.message);
+    it("Does not touch errors that are neither AmqpError or SystemError", function() {
```

Comment:
either neither and nor or either and or (so not neither and or or either and nor).

Reply:
I'll make the change, though this took me on a fun [journey](https://www.merriam-webster.com/dictionary/neither#usage-1) :).